### DISCUSSION TITLE
The screw

### DISCUSSION THREAD

**Discussion by wocard2** [ataradov/usb-sniffer#20] [General]
What screw fits the case?

2.2 mm DIN 7982 is too thin (doesn't stay in the hole), 2.9 mm DIN 7982 is too wide (will probably burst the case) and the diameter of the heads of both is too big for the cover.  The DIN 7982 series doesn't have any diameter between those two. So, it's probably not a DIN 7982 or DIN 7972 or ISO 14586 screw.

I'm not familiar with US screws...

**Comment by ataradov** [comment]
This is designed for 1.6 mm screw with a threaded heat-pressed insert.
The specific screws and inserts I used - https://www.aliexpress.us/item/2251832412304819.html and https://www.aliexpress.us/item/2251832703922707.html

**Comment by ataradov** [comment]
I think I used 6 mm or 8 mm screw length. I can't check right now.

**Comment by ataradov** [comment]
And I assume it is possible to assemble this with a small self tapping screw and a piece of wood jammed into the big hole.

**Comment by wocard2** [comment]
All the srew types I mentioned are self-tapping screws. I have no idea where the insert would go (unless the case is modified).
I'll try epoxy or other filling material and a smaller self-tapper from my stash of unlabeled screws, with a head that fits the cover.

**Comment by wocard2** [comment]
Some splinters of a match held in place with some epoxy and an M2 screw are good enough for me. Couldn't find self-tapping screws with a diameter smaller than 2.2 mm.

**Comment by ataradov** [comment]
The diameter of the hole in the main case body is designed for the outside diameter of the insert (2.7 or 2.8 mm). Insert is heated and pressed into the hole, which turns the plastic hole into a threaded M1.6 hole for a machine screw. 

The screw goes through the recessed hole in the cap.

I realize that having those inserts is not ideal for everyone, but I don't have any self tapping screws on hand and I also did not have a lot of success printing small diameter holes on my printer.

I'll get some standard self tapping screws and do more experiments.

**Comment by wocard2** [comment]
Now I understood, thanks! I found a local supplier for those inserts, but I don't know whether I'll dare to try that approach.

**Comment by ataradov** [comment]
They are very easy to use. Once heated with hot air station, they go into the plastic with no effort at all.

The only issue I see is that different printers have different tolerances, so the hole may end up being too loose for the insert. Too tight is generally not a problem, since plastic melts. I just tuned the hole diameter for my printer.

**Reply by wocard2** [reply]
This was much easier than anticipated, thanks! The hole indeed is a bit too wide (case manufactured by JLCPCB), I could pull out the insert if I really tried to.

**Comment by ataradov** [comment]
I've seen that PcbWay asks if there are any holes for standard inserts in the part. I'm not sure what they will do with this information. Possibly resize the hole according to their tolerances.
